### DISCUSSION TITLE
wildrift-gold-efficiency/5.1a/

### DISCUSSION THREAD

**Discussion by giscus** [changchiyou/wildrift-gold-efficiency#3] [Announcements]
# wildrift-gold-efficiency/5.1a/

Buff FORCE OF NATURE, RUIN - BLADE OF THE RUINED KING, THORNMAIL; adjust RUIN - INFINITY EDGE, RUIN - RABADON’S DEATHCAP; nerf TERMINUS, LIGHT - YOUMUU’S GHOSTBLADE

https://changchiyou.github.io/wildrift-gold-efficiency/5.1a/

**Comment by changchiyou** [comment]
[WILD RIFT PATCH NOTES 5.1A - ITEMS](https://wildrift.leagueoflegends.com/en-us/news/game-updates/wild-rift-patch-notes-5-1a/#items)

---

#### FORCE OF NATURE

We’re boosting the cost-efficiency of Force of Nature so it can become a more desirable pickup when you’re against an AP-heavy team.

-   Total Price<i>:</i> 2850g → **2750g**
-   Magic resistance: 50 → **55**
-   Absorb: Taking ability damage from enemy champions grants 1 stack of Steadfast, max 6 → **4** stacks. At maximum stacks, gain 10% Movement Speed and reduce all incoming magic damage by 25%.

#### RUIN - BLADE OF THE RUINED KING

The health reduction has been more impactful than intended, especially in the early game. We’re adjusting it so it’s still a valuable pickup, but toning it down just a bit.

-   Repentance: Reduces your base Health by 10 - 100 → **5 - 80**

#### RUIN - INFINITY EDGE

We're also adjusting the health reduction that Infinity Edge grants as it was stronger than we intended.

-   Self-Destruction: Reduces your base Health by 10 - 100 → **5 - 80**
-   Infinity: Critical Strikes deal 225% → **220%** damage instead of 175%

#### RUIN - RABADON’S DEATHCAP

We’re adjusting the numbers of Rabadon’s Deathcap to make it a more reasonable pick up.

-   Cinders: Reduces your base Health by 10-100 → **5-80**
-   Overkill: Increases Ability Power by 25-55% → **25-50%** (based on level)

#### THORNMAIL

We’ve increased the cost efficiency of Thornmail to make it a reliable item for pickup when you’re combating champions with a lot of Health Regen or Healing.

-   Maximum Health: 100 → **200**
-   Thorns: When struck by an enemy champion's basic attack, deal 20 + 6% bonus armor + 1% → **2%** bonus Health as magic damage to the attacker and inflict 50% Grievous Wounds for 3s

#### TERMINUS

We’re introducing an upper limit for the amount of Armor and Magic Penetration gained with Terminus.

-   Juxtaposition<b>: [New]</b> The upper limit of armor and magic penetration rates is 40% when equipped with Terminus

#### LIGHT - YOUMUU’S GHOSTBLADE

We’re adjusting the amount of Armor Penetration Youmuu’s grants so it’s a more desirable choice when facing a team that is stacking armor.

-   Momentum: Momentum grants up to 50 Movement Speed and 5-10 → 3-6 armor penetration at 100 stacks
